CRM/Utils/System/Drupal8.php

```diff
@@ -49,7 +49,7 @@ public function createUser(&$params, $mail) {
     if ($user_register_conf != 'visitors' && !$user->hasPermission('administer users')) {
       $account->block();
     }
-    elseif ($verify_mail_conf) {
+    elseif {
```

Comment:
```suggestion
    else {
```

Reply:
Pushed, you are really quick. I was about to push the change